codePuls.Application/Services/ProjectService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Extensions.Logging;
+using codePuls.Application.DTOs.RequestDTOs;
+using codePuls.Application.DTOs.ResponseDTOs;
+using codePuls.Application.Interfaces;
+using codePuls.Domain.Entities;
+using CodePuls.Domain.Interfaces.Repositories;
+using Mapster;
+using codePuls.Application.Exeptions;
+
+namespace codePuls.Application.Services
+{
+    public class ProjectService : IProjectService
+    {
+        private readonly IProjectRepository _projectRepository;
+        private readonly ILogger<ProjectService> _logger;
+
+
+        public ProjectService(IProjectRepository projectRepository, ILogger<ProjectService> logger) 
+        {
+            _projectRepository = projectRepository;
+            _logger = logger;
+        }
```

Comment:
use primary constructors which will remove all these line
```suggestion
    public class ProjectService(IProjectRepository projectRepository, ILogger<ProjectService> logger) : IProjectService
    {
```